loom/vendor/stb/stb_truetype.h

```diff
@@ -630,7 +630,7 @@ enum { // languageID for STBTT_PLATFORM_ID_MAC
 static int stbtt__isfont(const stbtt_uint8 *font)
 {
    // check the version number
-   if (stbtt_tag(font, "1"))   return 1; // TrueType 1
+   if (font[0] == '1')            return 1; // TrueType 1
```

Comment:
The correct version of this fix is still to check all four bytes; this is how the code in question appears in the current release:

```
if (stbtt_tag4(font, '1',0,0,0)) 
```
I don't remember how long ago this was fixed. You could also fix it as `stbtt_tag4(font, "1\0\0\0")`